be/CMakeLists.txt

```diff
@@ -625,6 +625,10 @@ if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
         set(CXX_COMMON_FLAGS "${CXX_COMMON_FLAGS} -Wno-switch-default")
     endif ()
 else ()
+    if (CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL "14.0.0")
+        # ignore error from apache-orc
+        set(CXX_COMMON_FLAGS "${CXX_COMMON_FLAGS} -Wno-error=dangling-reference")
```

Comment:
This will be effective for all the project files, feel like we shall have a refactor to set these to apache-orc target instead of global flags

Reply:
Currently still report warnings. will remove this flag when all warnings are resolved.